app/code/Magento/Theme/Block/Html/Footer.php

```diff
@@ -123,4 +122,14 @@ public function getIdentities()
     {
         return [\Magento\Store\Model\Store::CACHE_TAG, \Magento\Cms\Model\Block::CACHE_TAG];
     }
+
+    /**
+     * Get block cache life time
+     *
+     * @return int
+     */
+    protected function getCacheLifetime()
```

Comment:
Shouldn't it be public?

Reply:
hey @ihor-sviziev it is protected in \Magento\Framework\View\Element\AbstractBlock::getCacheLifetime

Reply:
Ah, ok. So I approved your changes